src/debug.c

```diff
@@ -3012,7 +3012,7 @@ static void DebugAction_Give_Item_SelectQuantity(u8 taskId)
 
         if (JOY_NEW(DPAD_UP))
         {
-            u32 maxCapacity = (ItemId_GetPocket(itemId) - 1 == BERRIES_POCKET) ? MAX_BERRY_CAPACITY : MAX_BAG_ITEM_CAPACITY;
+            u32 maxCapacity = (ItemId_GetPocket(itemId) - 1 == BERRIES_POCKET) ? MAX_BAG_ITEM_CAPACITY : MAX_BAG_ITEM_CAPACITY;
```

Comment:
```suggestion
            u32 maxCapacity = MAX_BAG_ITEM_CAPACITY;
```

Reply:
Kinda pointless to keep the local variable at all, isn't it?

```diff
lunos:/mnt/c/Users/Lunos/Home/rhh$ diff src/debug.c
diff --git a/src/debug.c b/src/debug.c
index e79e998ea..f4b324445 100644
--- a/src/debug.c
+++ b/src/debug.c
@@ -2876,10 +2876,9 @@ static void DebugAction_Give_Item_SelectQuantity(u8 taskId)

         if (JOY_NEW(DPAD_UP))
         {
-            u32 maxCapacity = (ItemId_GetPocket(itemId) - 1 == BERRIES_POCKET) ? MAX_BAG_ITEM_CAPACITY : MAX_BAG_ITEM_CAPACITY;
             gTasks[taskId].tInput += sPowersOfTen[gTasks[taskId].tDigit];
-            if (gTasks[taskId].tInput > maxCapacity)
-                gTasks[taskId].tInput = maxCapacity;
+            if (gTasks[taskId].tInput > MAX_BAG_ITEM_CAPACITY)
+                gTasks[taskId].tInput = MAX_BAG_ITEM_CAPACITY;
         }
         if (JOY_NEW(DPAD_DOWN))
         {
lunos:/mnt/c/Users/Lunos/Home/rhh$
```

Reply:
yeah, I realized it after the fact.

Reply:
Alright, then I'm greenlight to remove it? I kept it just in case 😆 

The same goes for the other local variables.

Reply:
yeah